cms/static/js/spec/views/move_xblock_spec.js

```diff
@@ -553,6 +546,111 @@ define(['jquery', 'underscore', 'edx-ui-toolkit/js/utils/spec-helpers/ajax-helpe
+                    var outline,
+                        libraryContentXBlockInfo = {
+                            category: 'library_content',
+                            display_name: 'Library Content',
+                            has_children: true,
+                            id: 'LIBRARY_CONTENT_ID'
+                        },
+                        outlineOptions = {library_content: 1, component: 1};
+
+                    // make above xblock source xblock.
+                    modal.sourceXBlockInfo = libraryContentXBlockInfo;
+                    outline = createXBlockInfo('component', outlineOptions, libraryContentXBlockInfo);
+                    renderViews(outline);
+                    expect(modal.$el.find('.modal-actions .action-move').hasClass('is-disabled')).toBeTruthy();
+
+                    // select a target parent
+                    clickForwardButton(0);
+                    expect(modal.$el.find('.modal-actions .action-move').hasClass('is-disabled')).toBeTruthy();
+                });
+
+                it('is disabled when navigating inside source content experiment', function() {
+                    var outline,
+                        splitTestXBlockInfo = {
+                            category: 'split_test',
+                            display_name: 'Content Experiment',
+                            has_children: true,
+                            id: 'SPLIT_TEST_ID'
+                        },
+                        outlineOptions = {split_test: 1, unit: 2, component: 1};
+
+                    // make above xblock source xblock.
+                    modal.sourceXBlockInfo = splitTestXBlockInfo;
+                    outline = createXBlockInfo('unit', outlineOptions, splitTestXBlockInfo);
+                    renderViews(outline);
+                    expect(modal.$el.find('.modal-actions .action-move').hasClass('is-disabled')).toBeTruthy();
+
+                    // navigate to groups level
+                    clickForwardButton(0);
+                    expect(modal.$el.find('.modal-actions .action-move').hasClass('is-disabled')).toBeTruthy();
+
+                    // navigate to component level inside a group
+                    clickForwardButton(0);
+
+                    // move should be disabled because we are navigating inside source xblock
+                    expect(modal.$el.find('.modal-actions .action-move').hasClass('is-disabled')).toBeTruthy();
+                });
+
+                it('is disabled when navigating to any content experiment groups', function() {
+                    var outline,
+                        splitTestXBlockInfo = {
+                            category: 'split_test',
+                            display_name: 'Content Experiment',
+                            has_children: true,
+                            id: 'SPLIT_TEST_ID'
+                        },
+                        outlineOptions = {split_test: 1, unit: 2, component: 1};
```

Comment:
actually group in content experiments are verticals/units, so this will add slit test and 2 units and a component in group 0